src/cloud-api-adaptor/test/e2e/assessment_helpers.go

```diff
@@ -181,12 +174,9 @@ func IsPulledWithNydusSnapshotter(ctx context.Context, t *testing.T, client klie
 				if nydusSnapshotterPullRegex.MatchString(line) {
```

Comment:
@stevenhorsman were you able to pass a test that uses this `IsPulledWithNydusSnapshotter()` method? Just want to double check the `nydusSnapshotterPullRegex` didn't change on main.

Reply:
Yes, it seems to be the same log messages, which is what we check